src/components/screens/ScoreSetEditor.vue

```diff
@@ -1291,7 +1313,6 @@ export default {
         this.geneName = null
         this.geneNameAccessionSuggestions = []
         this.geneNameDropdownValue =  null
-        this.targetGenes = []
```

Comment:
why is this deleted? what is the effect of not clearing this on clear()?

Reply:
It's repeated. Another `this.targetGenes = []` is in line 1308.

Reply:
this.targetGenes is no longer getting cleared in the 'New score set' case when clear is clicked due to this deletion. not sure what you mean by it's repeated here, I don't see it on l1308 before or after your changes.

Reply:
Maybe the repeated codes were from my other branch.